scripts/required-files/pre-configured_extra_vars.yml

```diff
@@ -0,0 +1,124 @@
+##
+# Whether to deploy sylva-core
+#
+deploy_sylva_core: false
+sylva_core_repo_url: https://gitlab.com/codefol/sylva-core.git
+sylva_core_branch: metal3_existing_rancher
+baremetal_repo_url: https://github.com/suse-edge/charts.git
+baremetal_branch: main
+
+# VM user name
+vm_user: metal
+
+# VM user plain text password (not hash)
+vm_user_plain_text_password: metal
+
+# NOTE: this should be *your* (local user) SSH public key since *you*
+# will be using it to login to the VMs. The SSH public keys listed
+# here will be appended to the VM user's authorized_keys file.
+#
+vm_authorized_ssh_keys:
+ -YOU CAN REPLACE THIS BUT THE SCRIPT WILL CHANGE THIS AUTOMATICALLY 
+ 
+rke2_channel_version: v1.24
+
+dns_domain: suse.baremetal
+
+
+metal3_provisioning_nic: &metal3_provisioning_nic eth1
+
+
+# metal3_vm_libvirt_network_params: '--network bridge=virbr0,model=virtio --network bridge=br-eth3,model=virtio'
+metal3_vm_libvirt_network_params: '--network bridge=m3-egress,model=virtio --network bridge=m3-prov,model=virtio'
+
+#vm_memory: 16384
```

Comment:
Should we uncomment this?

Reply:
It's already set elsewhere so it can be removed, though I was thinking about trying to move everything into the extra_vars.yml file instead of having to search around to change it.